rest_api/src/route_handlers/bonds.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use rocket::http::RawStr;
+use rocket::http::Status;
+use rocket::response::Failure;
+use rocket_contrib::{Json, Value};
+
+#[post("/bonds")]
+pub fn create_bond() -> Result<Json<Value>, Failure> {
```

Comment:
Consider returning `Result<Json<T>, Custom<Json>>`

Reply:
Updated to return a custom ApiError responder instead of Failure or Custom<Json>